_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
@nscyclone you're awesome, thank you for the help!  I'll pull this down today and take a peek! 👍